.github/workflows/report_quiz/report_quiz.sh

```diff
@@ -46,14 +47,36 @@ main(){
         # We suppose that at this point we will have at least 2 options valid
         options=$([ -z "$A" ] || echo $(escp "$A"))$([ -z "$B" ] || echo ,$(escp "$B"))$([ -z "$C" ] || echo ,$(escp "$C"))$([ -z "$D" ] || echo ,$(escp "$D"))
         echo $options
+        
+        # We verify if the options don't exceed the max length
+        max_length=100
```

Comment:
You can put this variable as global and in capital letter, there is no need to have it inside this function !